media-plugins/vdr-burn/vdr-burn-0.3.0-r3.ebuild

```diff
@@ -28,28 +32,16 @@ RDEPEND="${DEPEND}
 	media-video/transcode
 	sys-apps/util-linux"
 
-# depends that are not rdepend
-DEPEND="${DEPEND}
-		dev-libs/boost"
-
-S="${WORKDIR}/${P#vdr-}"
+PATCHES=( "${FILESDIR}"/${P}_gentoo-path.patch
+	"${FILESDIR}"/${P}_setdefaults.patch
+	"${FILESDIR}"/${P}_dmh-archive.patch )
+QA_FLAGS_IGNORED="
+	usr/lib/vdr/plugins/libvdr-.*
```

Comment:
This is a source package, do we really need to ignore *flags? Normaly it's used for binary packages?